estuary/api/v1.py

```diff
@@ -227,12 +235,17 @@ def _get_partial_stories(results, reverse=False):
 
     # Adding the artifact itself if its story is not available
     if not all_results:
+        base_instance = estuary.utils.story.BaseStoryManager()
+        wait_times, total_wait_time = base_instance.get_wait_times([item])
```

Comment:
Doesn't `get_wait_times` require 2 or more artifacts in the story?

Reply:
If there are less than two, it returns [0], 0